WorkflowTesting/Sources/RenderTesterResult.swift

```diff
@@ -129,13 +130,23 @@ extension RenderTesterResult where WorkflowType.State: Equatable {
     ///   and is expected to mutate it to match the new state.
     @discardableResult
     public func assertStateModifications(
-        file: StaticString = #file,
+        _ modifications: (inout WorkflowType.State) throws -> Void,
+        fileID: StaticString = #fileID,
+        filePath: StaticString = #filePath,
         line: UInt = #line,
-        _ modifications: (inout WorkflowType.State) throws -> Void
+        column: UInt = #column
```

Comment:
ah yeah maybe should probably create a new function and deprecate this one. Though it would be a lot easier to use v1.2.1 for now and handle upgrading later.